node/resolvers/search/index.ts

```diff
@@ -282,6 +288,42 @@ const getSellers = async (
   return result?.find((region: Region) => region.id === regionId)?.sellers
 }
 
+const buildSpecificationFiltersAsFacets = (specificationFilters: string[]): SelectedFacet[] => {
+  return specificationFilters.map((specificationFilter: string) => {
+    const [key, value] = specificationFilter.split(":")
+    return { key: key, value: value }
+  })
+}
```

Comment:
```suggestion
const buildSpecificationFiltersAsFacets = (specificationFilters: string[]): SelectedFacet[] =>
  specificationFilters.map((specificationFilter: string) => {
    const [key, value] = specificationFilter.split(":")
    return { key, value }
  })
```
since you are only doing a `return` within your arrow function, you can remove the body brackets and the `return`, what comes after the arrow will already be returned.

also, if you have variables with the same name as your object's keys, you don't need to set the keys again, so instead of `{ key: key, value: value }` you can use only `{ key, value }`.